src/reservations/reservations.controller.ts

```
@@ -0,0 +1,577 @@
+    @Param('reservationId', ParseIntPipe) reservationId: number,
+    @Body() updateDto: UpdateReservationRequestDto
+  ): Promise<UpdateReservationResponseDto> {
+     
+    return {
+      code: '200',
+      message: 'OK',
+      data: {
+        id: reservationId,
+        title: updateDto.title || '오아시스를 직접 본다니',
+        category: updateDto.category || '아구',
+        reservationDatetime: updateDto.reservationDatetime || '2025-08-21T19:00:00+09:00',
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)